health_cards.gemspec

```diff
@@ -5,16 +5,30 @@ require 'rake'
 Gem::Specification.new do |spec|
   spec.name          = 'health_cards'
   spec.version       = HealthCards::VERSION
-  spec.authors       = ['Reece Adamson']
+  spec.authors       = ['Reece Adamson',
+                        'Samuel Sayer',
+                        'Stephen MacVicar',
+                        'Neelima Karipineni',
+                        'Daniel Lee',
+                        "Mick O'Hanlon",
+                        'Priyank Madria',
+                        'Shaumik Ashraf']
   spec.email         = ['radamson@mitre.org']
-  spec.summary       = 'Create Health Cards using FHIR and Verifiable Credentials'
-  spec.description   = 'Create Health Cards using FHIR and Verifiable Credentials'
+  spec.summary       = <<~TEXT
+    Create SMART Health Cards using FHIR and Verifiable Credentials for secure and decentralized#{' '}
+    presentation of clinical information.
+  TEXT
+  spec.description = <<~TEXT
+    Health Cards implements SMART Health Cards, a secure and decentralized framework that allows#{' '}
```

Comment:
I think most of my comments from the README apply here as well

Reply:
fixed